app/assets/javascripts/timur.jsx

```diff
@@ -25,6 +25,7 @@ import {BrowserContainer as Browser} from './components/browser/browser';
 import {PlotterContainer as Plotter} from './components/plotter/plotter';
 import {MessagesContainer as Messages} from './components/messages';
 import {TimurNavContainer as TimurNav} from './components/timur_nav';
+import {HomePageContainer as HomePage} from './components/home_page';
```

Comment:
Rather than exporting 'HomePageContainer' and aliasing it here, we might do:

export const HomePage = React.createClass(...)

export default connect()(HomePage)

then, to get the unconnected widget:

import { HomePage } from './components/home_page';

but the default import gives you a connected widget:

import HomePage from './components/home_page';

Reply:
I like to explicitly import the container so I know what item I am dealing with.